### DISCUSSION TITLE
Resume building and ATS review

### DISCUSSION THREAD

**Discussion by AshutoshKumar0206** [recodehive/resume-pitch#83] [Q&A]
Please review my resume and suggest suitable changes.
[AshutoshKumar_Resume.pdf](https://github.com/user-attachments/files/17005207/AshutoshKumar_Resume.pdf)

**Comment by sanjay-kv** [comment]
[ash.pdf](https://github.com/user-attachments/files/17084466/ash.pdf)

[ATS Version of CV (19).docx](https://github.com/user-attachments/files/17084467/ATS.Version.of.CV.19.docx)
@AshutoshKumar0206
